### PR TITLE
Build /ai/consulting and remove /ai/services

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built page as per [doc](https://docs.google.com/document/d/1Zj9agFHVT5pEXapazCcoh_o2v0jEU1Oe7BmdvDbDGrg/edit) and [design](https://www.figma.com/file/sGeCjFvA1kV9Kmvg2S2VBn/23.10-U.com---AI-bubble?node-id=248%3A1446&mode=dev)
- Removed /ai/services as per instructions in doc
- Added redirect: /ai/services -> /ai/consulting
- **Note: The class for this suru hasn't been merged into the feature branch yet, hence why you don't see it here. When https://github.com/canonical/ubuntu.com/pull/13146 is merged the suru will show up as expected**

## QA

- View the site locally in your web browser at: https://ubuntu-com-13154.demos.haus/ai/consulting
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5662

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
only one change - I would surround this with hrs and drop the white background - white only works when at thevery bottom. cc @juanruitina 

For example, here's how I've done it on Ubuntu Pro:
image

**Comment by bartaz** [comment]
Seems that hrs around "Connect with an expert" are wider than the grid in other parts of the page:

image

**Comment by mtruj013** [comment]
> I would think the HR below the CTA button is redundant, given the section below already has it

In my non-designer opinion, I agree. Honestly I think the white deep strip looked fine as it was

**Comment by lyubomir-popov** [comment]
yes there shouldn't be two lines,m and theyhh should be same widths - see lots of examples on the ubuntu pro page https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?type=design&node-id=4091%3A20096&mode=design&t=wVK6tKRIjrDnUHS6-1

**Comment by mtruj013** [comment]
> yes there shouldn't be two lines,m and theyhh should be same widths

Done, thanks both!

**Comment by lyubomir-popov** [comment]
Sorry just noticed a few small things:

the top two sections can be improved a bit on medium screens:
image
(copy from the browser, apologies for the poor formatting):
```
<section class="p-strip--whitesuru is-shallow">
  <div class="row">
      <div class="col-start-large-4 col-9 col-medium-2">
        <div class="p-section--shallow">
          <h1>AI consulting</h1>
        </div>
      </div>
      <div class="col-start-large-4 col-9 col-medium-4">
        <p data-gtm-vis-has-fired-1014405_503="1">Taking projects from experimentation to production requires expertise in machine learning operations. Canonical’s AI consulting services support you every step of the way.</p>
        <ul class="p-list--divided">
              <li class="p-list__item is-ticked">Explore the most suitable use cases</li>
              <li class="p-list__item is-ticked">Identify improvement opportunities in the data collection process</li>
              <li class="p-list__item is-ticked">Design an AI stack suitable for your organisation</li>
              <li class="p-list__item is-ticked">Deploy your solution and run AI at scale</li>
        </ul>
        <a class="p-button js-invoke-modal" href="/ai/contact-us">Contact us</a>
      </div>
    </div>
</section>
<hr class="p-rule is-fixed-width">
<section class="p-section">
  <div class="row">
    
    <div class="col-start-large-4 col-9">
      <div class="p-section--shallow">
        <h2>Define your AI journey.<br>Move projects to production faster.</h2>
      </div>
      
      
      
    </div><div class="col-start-large-4 col-9 col-medium-4 col-start-medium-3">
      <p data-gtm-vis-has-fired-1014405_503="1">Taking projects from experimentation to production requires expertise in machine learning operations. Canonical’s AI consulting services support you every step of the way.</p>
      <ul class="p-list--divided">
          <li class="p-list__item is-ticked">Explore the most suitable use cases</li>
          <li class="p-list__item is-ticked">Identify improvement opportunities in the data collection process</li>
          <li class="p-list__item is-ticked">Design an AI stack suitable for your organisation</li>
          <li class="p-list__item is-ticked">Deploy your solution and run AI at scale</li>
        </ul>
      <a class="p-button js-invoke-modal" href="/ai/contact-us">Contact us</a>
    </div>
  </div>
</section>
```

We should put this in a vanilla example, as it shows how we handle the 25/75 on medium screens in two cases - very short main heading, and a long main heading. cc @bartaz @juanruitina 

-  can we add a br in this h2 and align it like so on medium screens:
image
on large screens the br should persist:
image

- The last p-section should always be deep (@bartaz maybe we ca naddd this to the documentation? deep p-section if same colour as  preceding section, deep p-strip if different (white)), so please replace this p-strip with a deep p-section:
image

File: templates/ai/consulting.html [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+          </div>
+        </li>
+      </ol>
+    </div>
+    <hr>
+  </div>
+</section>
+
+
+<section class="p-strip">
```
Comment (akbarkz) [inline templates/ai/consulting.html:236]: as for as the design is concerned, looks like this section has to be with the white background

Comment (lyubomir-popov) [reply templates/ai/consulting.html:236]: White is a way to highlight a main cta section - the preceding one in this case. It only works well when the CTA section is last. So in this case, it shouldn't be white, as there's not much use highlighting a resources section. We want people's attention on the preceding one. Again, something we should have in the vanilla documentation. cc @bartaz

File: templates/ai/consulting.html [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+      <p>Taking projects from experimentation to production requires expertise in machine learning operations. Canonical's AI consulting services support you every step of the way.</p>
+      <div class="p-section--shallow">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Explore the most suitable use cases</li>
+          <li class="p-list__item is-ticked">Identify improvement opportunities in the data collection process</li>
+          <li class="p-list__item is-ticked">Design an AI stack suitable for your organisation</li>
+          <li class="p-list__item is-ticked">Deploy your solution and run AI at scale</li>
+        </ul>
+      </div>
+      <a class="p-button js-invoke-modal" href="/ai/contact-us">Contact us</a>
```
Comment (akbarkz) [inline templates/ai/consulting.html:46]: in the design this button is called "Get in touch"

File: templates/ai/consulting.html (hunk 1 above)
Comment (akbarkz) [reply templates/ai/consulting.html:236]: @lyubomir-popov then why is it white in Figma?

Comment (lyubomir-popov) [reply templates/ai/consulting.html:236]: shouldn't be.

**Review by anthonydillon — APPROVED** [review]
LGTM with one small copy comment

**Pull request merged**